app/Http/Controllers/InviteController.php

```diff
@@ -24,7 +26,13 @@ public function send(SendAdminInviteMailRequest $request)
             'status' => AdminInvite::PENDING
         ]);
 
-        Mail::to($request->email)->send(new SendAdminInviteMail());
+
+        $unhashedPassword = Str::random(8);
+
+        $user = User::create(['name' => 'Administrator', 'email' => $request->email, 'password' => bcrypt($unhashedPassword), 'unhashedPassword' => $unhashedPassword]);
```

Comment:
I think the index ```unhashedPassword``` is not used when creating a new user.
So kindly remove it, thanks.